<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PRODUCTOS</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTabWidget" name="pantalla">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Sensor</string>
    </attribute>
    <widget class="QPushButton" name="bt_regresar">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Regresar</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tabla_sensor">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>221</width>
       <height>281</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <column>
      <property name="text">
       <string>ID_sensor</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="background">
       <color>
        <red>255</red>
        <green>0</green>
        <blue>191</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID_type</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="background">
       <color>
        <red>85</red>
        <green>255</green>
        <blue>127</blue>
       </color>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>BÃºsqueda</string>
    </attribute>
    <widget class="QPushButton" name="btn_buscar">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>BUSCAR</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_codigo">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="txt">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tabla_busqueda">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>211</width>
       <height>231</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <column>
      <property name="text">
       <string>ID_sensor</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="background">
       <color>
        <red>255</red>
        <green>0</green>
        <blue>191</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID_type</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="background">
       <color>
        <red>85</red>
        <green>255</green>
        <blue>127</blue>
       </color>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="bt_regresar_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Regresar</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Borrar</string>
    </attribute>
    <widget class="QLineEdit" name="lbl_codigo_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="txt_dos">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ok">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_borrar">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>BORRAR</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tabla_borrar">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>211</width>
       <height>231</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <column>
      <property name="text">
       <string>ID_sensor</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="background">
       <color>
        <red>255</red>
        <green>0</green>
        <blue>191</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID_type</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="background">
       <color>
        <red>85</red>
        <green>255</green>
        <blue>127</blue>
       </color>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>-10</y>
     <width>641</width>
     <height>421</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../PROYECTO_EPISW/recursos.qrc">:/newPrefix/Imagenes/images.jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>pantalla</zorder>
 </widget>
 <resources>
  <include location="../PROYECTO_EPISW/recursos.qrc"/>
 </resources>
 <connections/>
</ui>
